submodules/LIB/hardware/syn/genus/build.mk

```diff
@@ -0,0 +1,61 @@
+# (c) 2022-Present IObundle, Lda, all rights reserved
+#
+# This makefile segment is used at build-time
+#
+
+REMOTE_BUILD_DIR=$(USER)/$(BUILD_DIR_NAME)
+REMOTE_SYN_DIR=$(REMOTE_BUILD_DIR)/hardware/syn
+
+
+#include local syn segment
+ifneq ($(wildcard syn_build.mk),)
+include syn_build.mk
+endif
+
+OUTPUT_DIR=results
+
+#default node
+NODE ?= umc130
+
+SYN_SERVER=$(CADENCE_SERVER)
```

Comment:
This part should not be repeated for each tool. See simulation.

Reply:
See the FPGA folder, this is wrong . This way you need to repeat these commands for each syn tool.

Reply:
OK will change, I done it like this because other sythesiser's maybe won't need a cadence server.

Reply:
This PR shows it was deleted from the makefile but it should not have been deleted

Why pass that code to the build.mk since it is not specific of the synt tool?


Close this PR and do a new one pls.